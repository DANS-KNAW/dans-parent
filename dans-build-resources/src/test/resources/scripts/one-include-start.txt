#include <multi-line.txt>

Only one include, at the start of file.