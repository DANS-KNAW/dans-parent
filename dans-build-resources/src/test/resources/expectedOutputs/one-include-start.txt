line 1
line 2
line 3


Only one include, at the start of file.